packages/@aws-cdk/aws-codepipeline/lib/pipeline.ts

```diff
@@ -388,44 +386,16 @@ export class Pipeline extends PipelineBase {
 
   /**
    * Gets the role used for this action,
-   * including handling the case when the action is supposed to be cross-region.
+   * including handling the case when the action is supposed to be cross-account.
    *
    * @param stage the stage the action belongs to
    * @param action the action to return/create a role for
+   * @param actionScope the scope, unique to the action, to create new resources in
    */
-  private getRoleForAction(stage: Stage, action: IAction, actionScope: Construct): iam.IRole | undefined {
+  private roleUsedForAction(stage: Stage, action: IAction, actionScope: Construct): iam.IRole | undefined {
```

Comment:
weird name. getRoleForAction made more sense

Reply:
I changed it because I felt it was too close to the new method `getRoleFromActionPropsOrGenerateIfCrossAccount`. Changed back.